xslt/geonetwork_update_backlogs_324/transform-mcp-2.0.xsl

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mcp="http://schemas.aodn.org.au/mcp-2.0"
+                xmlns:gco="http://www.isotc211.org/2005/gco"
+                xmlns:gmd="http://www.isotc211.org/2005/gmd"
+                xmlns:gmx="http://www.isotc211.org/2005/gmx"
+                xmlns:geonet="http://www.fao.org/geonetwork"
+
+                exclude-result-prefixes="xsl mcp gco gmd gmx geonet"
+>
+
+
+    <xsl:template match="node()|@*">
+        <xsl:copy>
+            <xsl:apply-templates select="node()|@*"/>
+        </xsl:copy>
+    </xsl:template>
+
+    <xsl:template match="gmd:URL/text()[.='http://i.creativecommons.org/l/by/4.0/88x31.png']">https://licensebuttons.net/l/by/4.0/88x31.png</xsl:template>
```

Comment:
Need to change references in schema plugins e.g. https://github.com/aodn/schema-plugins/blob/d7a228297949c0c4eea115705ae7343de48316e3/iso19139.mcp-2.0/loc/eng/strings.xml#L23 and probably also need to push this change up to geonetwork repositories

imosmest is a little more problematic as we don't have the source code for that apart from whats on the server - I'd just YOLO it there.

@pblain - does this require a separate issue?